src/FileSystemProviders.fs

```diff
@@ -76,18 +81,26 @@ let private watchDir (directoryInfo: DirectoryInfo) =
 
     watcher
 
-[<TypeProvider>]
-type RelativeFileSystemProvider(config: TypeProviderConfig) as this =
-    inherit TypeProviderForNamespaces(config)
+[<Interface>]
+[<System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)>]
```

Comment:
I never used this attribute in the past.

Wouldn't marking the `type` as `private` do the same?